cypress/e2e/Colonnes/TableauStudies.cy.ts

```diff
@@ -0,0 +1,74 @@
+/// <reference types="Cypress" />
+import '../../support/commands';
+
+beforeEach(() => {
+  cy.login();
+});
+
+describe('Page des études - Colonnes du tableau', () => {
```

Comment:
```suggestion
describe('Page Studies - Colonnes du tableau', () => {
```
Juste une proposition comme plus haut tu écris Data Exploration comme dans le portail.

Reply:
Plus loin du écris en français fichier, des variants, ... du coup je ne sais pas la pertinence de ce commentaire lol à voir et appliquer partout ce qui te semble le plus juste.